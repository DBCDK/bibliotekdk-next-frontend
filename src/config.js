/**
 * @file
 * Application configuration.
 */

const config = {
  port: process.env.PORT || 3000,
};

export default config;
