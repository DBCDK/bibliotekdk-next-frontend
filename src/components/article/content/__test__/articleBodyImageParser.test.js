import parseArticleBody from "../utils";
import data from "./__fixtures__/article.mock";
import config from "@/config";

test("parseArticleBodyImages", () => {
  const actual = parseArticleBody(data.article.body.value);
  expect(actual).toMatchSnapshot();
});

test("replaceString", () => {
  const replaceme =
    "fisk http://bibdk-backend-www-master.frontend-prod.svc.cloud.dbc.dk/ hest og køer";
  const expected = "fisk " + config.backend.url + "/ hest og køer";
  const actual = parseArticleBody(replaceme);

  expect(actual).toEqual(expected);
});
