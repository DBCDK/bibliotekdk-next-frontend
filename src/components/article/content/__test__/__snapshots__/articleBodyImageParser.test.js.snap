// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`parseArticleBodyImages 1`] = `"<h2>Overskrift 2</h2><p>Lorem ipsum dolor sit amet, consectetur adipiscing elit</p><figure> <img src=\\"/_next/image?url=/img/doppler.jpeg&w=1920&q=75\\" alt=\\"Doppler\\" title=\\"Erlend Loe - Doppler\\"> <figcaption>Erlend Loe - Doppler</figcaption></figure><p>Pellentesque varius tincidunt egestas. Vivamus rutrum accumsan arcu in auctor.</p>"`;
